equiTEA/theTEA
# adds app service; composes marketing and app services

- Composes a dev environment with docker-compose.dev.yml which reads Dockerfile.dev in the `marketing/` and `app/` directories.

## To Test:
1) Pull down the branch
2) Either run `docker-compose -f docker-compose.dev.yml up [--build]` or run the provided `dev.sh` shortcut from the root of the project
3) See the two Next.js apps running simultaneously: `localhost:3000` (marketing) and `localhost:3001` (app – currently an empty project)
